test/parallel/test-listen-fd-cluster.js

```diff
@@ -99,7 +99,16 @@ function master() {
     args: [ 'worker' ]
   });
   var worker = cluster.fork();
-  console.log('%j\n', { master: process.pid, worker: worker.pid });
+  worker.on('message', function(msg) {
+    if (msg === 'worker ready') {
+      process.send('started worker');
+    }
+  });
+  // Prevent outliving our parent process
```

Comment:
@bnoordhuis that comment was a bit brief... do you mean this code should not occur under normal conditions?

This is to prevent leaks in the case of an error/abnormal termination of the test script.

For example, if the test times out  then the top-level process, `parent()`, will get killed by the test runner, and its child process, `master()`, will leak. This prevents this. Note that the workers won't leak, because cluster has effectively the code above.